find_cmake_package_name.py

```diff
@@ -40,7 +37,7 @@ def parse_args():
         '--conan-root',
         metavar='conan_root',
         nargs=1,
-        default='{}/.conan/data'.format(environ['HOME']),
+        default=f'{environ["HOME"]}/.conan/data',
```

Comment:
The `environ["HOME"] lookup should rarely fail but if it does this will raise a `KeyError`. Either wrap this block in a `try/except` or pre-emptively check the success like so:

```
home = environ.get("HOME")
if home is not None:
    # Your error handling here
```

Reply:
Addressed.